lms/templates/support/certificates.html

```diff
@@ -9,8 +12,8 @@
 <%block name="js_extra">
 <%static:require_module module_name="support/js/certificates_factory" class_name="CertificatesFactory">
     new CertificatesFactory({
-        userFilter: '${ user_filter }',
-        courseFilter: '${course_filter}'
+        userFilter: '${ user_filter | n, js_escaped_string}',
+        courseFilter: '${ course_filter | n, js_escaped_string }'
```

Comment:
Nit: Drop surrounding spaces.  At this time it is inconsistent in userFilter as well.
Use:
```
 userFilter: '${user_filter | n, js_escaped_string}',
 courseFilter: '${course_filter | n, js_escaped_string}'
```